script_facade/api/fhir.py

```diff
@@ -22,12 +22,15 @@ def medication_order(fhir_version):
     patient_lname = request.args.get('subject:Patient.name.family', default_patient_lname)
     patient_dob = request.args.get('subject:Patient.birthdate', default_patient_dob).split('eq')[-1]
 
+    script_version = request.args.get('script_version')
```

Comment:
we only support two, looks like case would matter, etc.
I'd suggest validation of `script_version`